File: LightOJ/LightOJ1000.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin >> t;
	for(int i = 1; i <= t; i++)
	{
		int a, b;
		cin >> a >> b;
		int sum = a + b;
		cout << "Case " << i << ": " << sum << endl;
	}
	return 0;
}

File: LightOJ/LightOJ1000.java
import java.util.Scanner;
public class Main
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here\
		Scanner sc = new Scanner(System.in);
		int test = sc.nextInt();
		for(int i = 1; i <= test; i++)
			System.out.println("Case " + i + ": " + (sc.nextInt() + sc.nextInt()));
	}
}